source/jormungandr/tests/bragi_autocomplete_tests.py

```diff
@@ -90,7 +95,8 @@ def test_autocomplete_call(self):
                 }, 200)
         })
         with mock.patch('requests.get', mock_requests.get):
-            response = self.query_region('places?q=bob&pt_dataset=main_autocomplete_test')
+            response = self.query_region("places?q=bob&pt_dataset=main_autocomplete_test&type[]=['stop_area', "
```

Comment:
`type[]=stop_area&type[]=adress` does not work in the tests as `requests` expects to have type[]=['stop_area', adress']

but `type[]=stop_area&type[]=adress` works perfectly from a browser